Tests/Integration/MSFT_xWebBindingInformation.Tests.ps1

```diff
@@ -1,59 +1,83 @@
+        It 'Should be able to get xWebsite' -test {
+            # just a good idea.
+            # I thought it might force the classes to register, but it does not.
+            $resources = Get-DscResource -Name xWebsite
+            $resources.count | should be 1
+        }
 
-    It 'Should compile and run without throwing' -test {
-        {
-        # Force Cim Classes to register
-        # Update the system environment path so that LCM will load the module
-        # Requires WMF 5
-        [System.Environment]::SetEnvironmentVariable('PSModulePath',$env:PSModulePath,[System.EnvironmentVariableTarget]::Machine)
-        configuration foo
-        {
-            Import-DscResource -ModuleName xWebAdministration
-
-            xWebsite foo
+        It 'Should compile and run without throwing' -test {
             {
-                Name = 'foobar'
-                Ensure = 'absent'
-                PhysicalPath = "$env:temp\foo"
-            }
+            # Force Cim Classes to register
+            # Update the system environment path so that LCM will load the module
+            # Requires WMF 5
+            [System.Environment]::SetEnvironmentVariable('PSModulePath',$env:PSModulePath,[System.EnvironmentVariableTarget]::Machine)
+
+            WebBindingInfo -OutputPath $env:temp\WebBindingInfo
+            Start-DscConfiguration -Path $env:temp\WebBindingInfo -Wait -Verbose -ErrorAction Stop} | should not throw
         }
 
-        foo -OutputPath $env:temp\foo
-        Start-DscConfiguration -Path $env:temp\foo -Wait -Verbose -ErrorAction Stop} | should not throw
-    }
-    
-    # Directly interacting with Cim classes is not supported by PowerShell DSC
-    # it is being done here explicitly for the purpose of testing. Please do not
-    # do this in actual resource code
-    $xWebBindingInforationClass = (Get-CimClass -Namespace 'root/microsoft/Windows/DesiredStateConfiguration' -ClassName 'MSFT_xWebBindingInformation')
-    $storeNames = (Get-CimClass -Namespace 'root/microsoft/Windows/DesiredStateConfiguration' -ClassName 'MSFT_xWebBindingInformation').CimClassProperties['CertificateStoreName'].Qualifiers['Values'].Value
-    foreach ($storeName in $storeNames){
-        It "Uses valid credential store: $storeName" {
-            (Join-Path -Path Cert:\LocalMachine -ChildPath $storeName) | Should Exist
+        # Directly interacting with Cim classes is not supported by PowerShell DSC
+        # it is being done here explicitly for the purpose of testing. Please do not
+        # do this in actual resource code
+        $xWebBindingInforationClass = (Get-CimClass -Namespace 'root/microsoft/Windows/DesiredStateConfiguration' -ClassName 'MSFT_xWebBindingInformation')
+        $storeNames = (Get-CimClass -Namespace 'root/microsoft/Windows/DesiredStateConfiguration' -ClassName 'MSFT_xWebBindingInformation').CimClassProperties['CertificateStoreName'].Qualifiers['Values'].Value
+        foreach ($storeName in $storeNames){
```

Comment:
Just a style question: Should the ```{``` on a ```foreach\for``` be on the next line like on an ```if``` statement?